src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        int index = random.nextInt(Colors.values().length);
+        return Colors.values()[index].toString();
```

Comment:
Use `name()` instead of `toString()` for getting the String representation of enum constants to ensure that you always get the exact name of the enum constant.